packages/stack/src/TransitionConfigs/TransitionPresets.tsx

```diff
@@ -150,3 +151,27 @@ export const ModalTransition = Platform.select({
   ios: ModalPresentationIOS,
   default: BottomSheetAndroid,
 });
+
+/**
+ * Slide from left transition.
+ */
+export const SlideFromLeftIOS: TransitionPreset = {
+  ...SlideFromRightIOS,
+  cardStyleInterpolator: forHorizontalIOSInverted,
+};
+
+/**
+ * Animation presets for the transitions.
+ */
+export const AnimationTransitions = {
+  default: DefaultTransition,
+  fade: ModalFadeTransition,
+  fade_from_bottom: FadeFromBottomAndroid,
+  none: DefaultTransition,
+  modal_slide_from_bottom: ModalSlideFromBottomIOS,
+  reveal_from_bottom: RevealFromBottomAndroid,
+  scale_from_center: ScaleFromCenterAndroid,
+  slide_from_bottom: BottomSheetAndroid,
+  slide_from_left: SlideFromLeftIOS,
+  slide_from_right: SlideFromRightIOS,
+} as const;
```

Comment:
I think this will ensure that all the animation names are kept in sync.

```suggestion
as const satisfies Record<StackAnimationTypes, TransitionPreset>;
```